tests/lib/prepare-restore.sh

```diff
@@ -304,6 +304,37 @@ prepare_project() {
         fi
     fi
 
+    # debian packaing is special
+    if [[ "$SPREAD_SYSTEM" == debian-sid* ]]; then
+        if [ ! -d packaging/debian-sid ]; then
+            echo "no packaging/debian-sid/ directory "
+            echo "broken test setup"
+            exit 1
+        fi
+
+        # remove etckeeper
+        apt purge -y etckeeper
+
+        # debian has its own packaging
+        rm -f debian
+        # the debian dir must be a real dir, a symlink will make
+        # dpkg-buildpackage choke later.
+        mv packaging/debian-sid debian
+
+        # get the build-deps
+        apt build-dep -y ./
+
+        # and ensure we don't take any of the vendor deps
+        rm -rf vendor/*/
+
+        # and create a fake upstream tarball
+        tar -c -z -f ../snapd_"$(dpkg-parsechangelog --show-field Version|cut -d- -f1)".orig.tar.gz --exclude=./debian --exclude=./.git .
```

Comment:
I was thinking about this step lately. I think we can merge this as-is but I would like to promote the act of making a release tarball into something that we have proper tools for, right in the tree. That is, that the tarball that we make on release day is the same tarball we were making in CI all those days before that.

Reply:
Right, currently we just use the tarfile generated with "gbp buildpackage -S" from the release branch.